frontend/simple/js/state.js

```diff
@@ -17,7 +16,15 @@ import debounce from 'lodash/debounce'
 Vue.use(Vuex)
 var store // this is set and made the default export at the bottom of the file.
           // we have it declared here to make it accessible in mutations
-
+// Used by the getters and mutations to find the mailbox contracts
+function getMailbox (state) {
+  for (let [key, value] of Object.entries(state.contracts)) {
+    if (value === 'MailboxContract') {
+      return key
+    }
+  }
+  return null
+}
```

Comment:
It may be a good idea to do this now. E.g. rename the function to `mailboxForUser`, and if it's named that, then it should also return a *mailbox* (not mailbox id, per [earlier comment](https://github.com/okTurtles/group-income-simple/pull/225/files#r110562104)).